labelCloud/control/controller.py

```diff
@@ -315,3 +315,14 @@ def key_release_event(self, a0: QtGui.QKeyEvent) -> None:
         if a0.key() == QtCore.Qt.Key_Control:
             self.ctrl_pressed = False
             self.view.status_manager.clear_message(Context.CONTROL_PRESSED)
+
+    def crop_pointcloud_inside_active_bbox(self) -> None:
+        box = self.bbox_controller.get_active_bbox()
+        assert box is not None
```

Comment:
The assertion is for mypy. You are right that this will never be the case as we only trigger it when we know there is an active box. Alternatively we can also return when box is None, but it is also redundant.